### PR TITLE
Droped to using seconds when waiting for reponses instead of the minute

### PR DESCRIPTION
**Pull request by srugano**

### CODE REVIEW

**Comment by srugano** [comment]
> It's missing Python and GoFlow changes for this PR

I added them on the floweditor [here](https://github.com/greatnonprofits-nfp/floweditor/pull/1).

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     ]
   },
   "dependencies": {
-    "@nyaruka/flow-editor": "1.6.2",
+0   "@nyaruka/flow-editor": "github:greatnonprofits-nfp/floweditor#current-ccl-version",
```
Comment (teehamaral) [inline package.json]: @srugano - This `0` should be removed

File: static/coffee/flows/controllers.coffee [hunk 2]
```diff
@@ -1056,27 +1056,31 @@ NodeEditorController = ($rootScope, $scope, $modalInstance, $timeout, $log, Flow
...
-    {value:120, text:'2 hours'},
-    {value:180, text:'3 hours'},
-    {value:360, text:'6 hours'},
-    {value:720, text:'12 hours'},
-    {value:1080, text:'18 hours'},
-    {value:1440, text:'1 day'},
-    {value:2880, text:'2 days'},
-    {value:4320, text:'3 days'},
-    {value:10080, text:'1 week'},
+    {value:10, text:'10 seconds'},
```
Comment (teehamaral) [inline static/coffee/flows/controllers.coffee]: @srugano - GoFlow it's waiting for minutes (https://github.com/greatnonprofits-nfp/goflow/blob/ccl-current-version/legacy/definition.go#L676), so in your implementation the values are wrong, that is, 10 seconds should be ~0.16, 20 seconds should be ~0.33, and so on.

**Comment by teehamaral** [comment]
@srugano - Should I close this PR because we already have open this another one --> #29 ?

**Pull request closed without merging**